extensions/multimodal/README.md

```diff
@@ -13,6 +13,8 @@ https://user-images.githubusercontent.com/3718215/233817203-69b57e77-0c55-4fd6-b
 To run this extension, download a LLM that supports multimodality, and then start server.py with the appropriate `--multimodal-pipeline` argument. Examples:
 
 ```
+# due to dimension mismatch, currently there are two versions of llava-v1.5 pipelines
+python server.py --model liuhaotian_llava-v1.5-7b --multimodal-pipeline llava-v1.5-7b --load-in-4bit
```

Comment:
Can we put 7B after 13B, as according to our eval, 13B generates outputs with a better quality than 7B?

Reply:
Done!